src/parser/README.md

```diff
@@ -1,33 +1,41 @@
-# The Flow Parser
+# The flow-parser package
 
-The Flow Parser is a JavaScript parser written in OCaml. It produces an AST that conforms to [SpiderMonkey's Parser API](https://developer.mozilla.org/en-US/docs/Mozilla/Projects/SpiderMonkey/Parser_API) and that mostly matches what [esprima](http://esprima.org/) produces. The Flow Parser can be compiled to native code or can be compiled to JavaScript using [js_of_ocaml](http://ocsigen.org/js_of_ocaml/).
+This package contains the Flow parser in its compiled-to-JavaScript form.
 
-## Building the Flow Parser
+# What is Flow
 
-Building the Flow Parser requires OCaml. Compiling to JavaScript requires js_of_ocaml >= 2.8.
+See [flowtype.org](http://flowtype.org/). The code for the Flow parser [lives on GitHub](https://github.com/facebook/flow/tree/master/src/parser).
 
-### Initial set up
+# What is the Flow Parser
```

Comment:
git is messing up the rename detection. this file was the old `README-npm.md` :/